api/models.py

```diff
@@ -38,53 +146,55 @@ def get_update_url(self):
         return reverse('api_station_update', args=(self.id,))
 
 
-class Metering(models.Model):
-
-    # Fields
-    created = models.DateTimeField(auto_now_add=True)
-    pm01 = models.FloatField()
-    pm25 = models.FloatField()
-    pm10 = models.FloatField()
-    temp_out1 = models.FloatField()
-    temp_out2 = models.FloatField(max_length=100)
-    temp_out3 = models.FloatField()
-    hum_out1 = models.FloatField()
-    hum_out2 = models.FloatField()
-    hum_out3 = models.FloatField()
-    temp_int1 = models.FloatField()
-    hum_int1 = models.TextField(max_length=100)
-    rssi = models.FloatField()
-    bpress_out1 = models.FloatField()
+class Metering(MeteringBase):
+    """
+    Model representing data submitted by sensor station.
+    """
 
-    # Relationship Fields
-    station = models.ForeignKey('api.Station', )
+    is_test = models.BooleanField(
+        help_text='Whether entry was created using Sensor test mode (Sensor.is_in_test_mode=True).',
+        default=False,
+        db_index=True
+    )
 
     class Meta:
         ordering = ('-created',)
 
-    def __unicode__(self):
-        return u'%s' % self.id
-
     def get_absolute_url(self):
-        return reverse('api_metering_detail', args=(self.id,))
+        return reverse('api_metering_detail', args=(self.created,))
 
-    def get_update_url(self):
-        return reverse('api_metering_update', args=(self.id,))
 
+class MeteringHistory(MeteringBase):
```

Comment:
@zen @grillazz Now I'm considering changing this into `HistoryMetering` instead... What do You think?